### DISCUSSION TITLE
[@grafana/ui] Allow users to re-arrange elements on a MultiSelect

### DISCUSSION THREAD

**Discussion by kminehart** [grafana/grafana#38161] [Dashboards, Panels & Visualizations]
**What would you like to be added**:

Some users have requested that they would like to be able to re-arrange the elements in a MultiSelect / AsyncMultiSelect.

![image](https://user-images.githubusercontent.com/5140827/95998741-cac7aa00-0dfa-11eb-8394-ce7b7f1ea420.png)

**Why is this needed**:

When they need to re-arrange the elements they have resorted to editing the panel JSON directly.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
